docs/configuration/trap-configuration.md

```diff
@@ -143,4 +143,10 @@ Example:
 ```yaml
 traps:
   aggregateTrapsEvents: "true"
+```
+
+### Updating trap configuration
+If you need to update some configuration regarding traps, you can do it by editing the `value.yaml` and then running below command to restart the pod deployment.
+```
+microk8s kubectl rollout restart deployment snmp-splunk-connect-for-snmp-trap -n sc4snmp
```

Comment:
I think name of the pod depends on helm name, most probably the pattern is <chart-app-name>-splunk-connect-for-snmp-trap, maybe it's worth mentioning.